src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -710,7 +710,9 @@ Semantics.ConversionKind IConversionExpression.ConversionKind
             }
         }
 
-        bool IConversionExpression.IsExplicit => this.ExplicitCastInCode;
+        public IConversion Conversion => new Conversion(this.ConversionKind, this.SymbolOpt, isExtensionMethod: false);
+
+        bool IConversionExpression.ExplicitCastInSource => this.ExplicitCastInCode;
```

Comment:
Note: i picked a different name for the IConversionExpressoin property because we already use "InSource" in other parts of that API to indicate this concept.  We might want to rename the internal property to match.  If anyone wants me to, i'm happy to do it.